h/static/styles/tags-input.scss

```diff
@@ -101,3 +101,47 @@ tags-input {
   }
 }
 
+tags-input .autocomplete {
```

Comment:
I'd like to see this whole section attempted with ems for the paddings, margins and font sizes. I'd like to see variables for the font families from our variables.scss. If you're going to use `1px` for a border just say `thin`.